### ISSUE TITLE
Throws on excluded files 

### ISSUE THREAD

**Issue by shvgn** [thorn0/organize-imports-cli#2]
Hi. 

When I do `yarn organize-imports-cli src/**/*.ts` or add it to git hook, it fails with error:

```
/Users/shvgn/********/node_modules/organize-imports-cli/cli.js:99
    logger.write(chalk`{gray ${sourceFile.getFilePath()}}`);
```

<details>
  <summary>Full stacktrace</summary>

```
/Users/shvgn/********/node_modules/organize-imports-cli/cli.js:99
    logger.write(chalk`{gray ${sourceFile.getFilePath()}}`);
                                          ^

TypeError: Cannot read property 'getFilePath' of undefined
  at main (/Users/shvgn/********/node_modules/organize-imports-cli/cli.js:99:45)
  at Object.<anonymous> (/Users/shvgn/********/node_modules/organize-imports-cli/cli.js:20:3)
  at Module._compile (internal/modules/cjs/loader.js:776:30)
  at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
  at Module.load (internal/modules/cjs/loader.js:653:32)
  at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
  at Function.Module._load (internal/modules/cjs/loader.js:585:3)
  at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
  at startup (internal/bootstrap/node.js:283:19)
  at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```
 
</details>

What I found out iin my specific case is that method `project.getSourceFile(filePath)` does not return SourceFile object if filename is `anything.spec.ts` that match the glob above. Here is the line where `undefined` is returned:

https://github.com/thorn0/organize-imports-cli/blob/master/cli.js#L93

My tsconfig

```
{ 
  ...
  "exclude": ["node_modules", "**/*.spec.ts", "test"]
}
```

The problem can be fixed if I remove *spec.ts from exclusions.

```
{
    ...
    "exclude": ["node_modules",  "test"]
}
```

From the prospect of TS, they are not in the project. Anyway `organize-imports-cli` processes them. I have a bunch of unit-tests within my modules, but they are excluded. I don't want test files to compile and include in build. They only run with mocha and ts-node. 

I think it is worth to catch the error do something. Maybe show a warning about a file not being in project or what the real reason is. I did not dig in ts-morph and TS compiler API to decide better.

**Comment by thorn0** [comment]
> organize-imports-cli src/**/*.ts

This doesn't work because globs aren't supported.

**Comment by shvgn** [comment]
Ok. Anyway the glob is not passed to the script directly. Shell expanded this glob into the list of files. Resulting command was like this one. I should have been more explicit on this.

```
organize-imports-cli src/file.ts src/file.spec.ts ...
```

Also I added logging to the line cli.js#L93. That was how saw script iterating over project files and that `getSourceFile` returns undefined for ones excluded in tsconfig.

**Comment by thorn0** [comment]
Yes, there was an issue with `exclude` indeed. Just published v0.6.0. Give it a try.
